### DISCUSSION TITLE
Ways to display Combo Contents

### DISCUSSION THREAD

**Discussion by DRBAUS** [RomanKubiak/ctrlr#252] [Q&A]
Is there a way to set how the Combo Contents are being displayed?

I have a list of 168 Waveforms that preferably I would like to display in 4 columns of 42 instead of 1 column of 168.

I think I have seen this in one of the Panels but can't remember which one.

If not possible what do you suggest otherwise?

**Comment by dnaldoog** [comment]
Hi Baus,

The best way to do that is with the class **PopupMenu()**. See my JV1010 panel.

https://ctrlr.org/jv-1010-editor/

**Reply by DRBAUS** [reply]
Thanks dnaldoog! That's the one that I saw.

It looks complicated so I will have to study that.
